iosApp/iosApp/Localizable.xcstrings

```diff
@@ -11339,6 +11339,7 @@
       }
     },
     "Open for more arrivals" : {
+      "extractionState" : "stale",
```

Comment:
This string was only used in the StopDeparturesSummaryList, although it’s still used in Android, so there’s probably a place it still belongs on iOS.

Reply:
I hear "Open for more arrivals" on VoiceOver when I hover on a row in the old route card. If not trivial to add here, can you make a follow-up for it?

Reply:
Added here.